miden/src/cli/bundle.rs

```diff
@@ -40,19 +39,19 @@ impl BundleCmd {
                 .into_owned(),
         };
 
-        let source_manager = Arc::new(assembly::DefaultSourceManager::default());
+        let assembler = Assembler::default().with_debug_mode(true);
         let library_namespace =
             namespace.parse::<LibraryNamespace>().expect("invalid base namespace");
         // TODO: Add version to `Library`
-        let _version = self.version.parse::<Version>().expect("invalid cargo version");
-        let stdlib = Library::from_dir(&self.dir, library_namespace, source_manager)?;
+        // let version = self.version.parse::<Version>().expect("invalid cargo version");
+        let library = Library::from_dir(&self.dir, library_namespace, assembler)?;
```

Comment:
If `bundle` only creates a `Library`, what's the canonical way of creating a `KernelLibrary`? Should we add a `--kernel` flag to this command (in another PR)?

Reply:
https://github.com/0xPolygonMiden/miden-vm/issues/1447